content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -186,7 +186,7 @@ containers:
 [1]: /agent/cluster_agent/admission_controller/
 [2]: https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/#capabilities-of-the-downward-api
 [3]: /agent/kubernetes/data_collected/#kube-state-metrics
-[4]: https://github.com/DataDog/integrations-core/blob/master/kubernetes_state/datadog_checks/kubernetes_state/data/conf.yaml.example#L70
+[4]: https://github.com/DataDog/integrations-core/blob/42ecba8180fdce32c2e6739e8db006da2941c29d/kubernetes_state/datadog_checks/kubernetes_state/data/conf.yaml.example#L63
```

Comment:
```suggestion
[4]: https://github.com/DataDog/integrations-core/blob/master/kubernetes_state/datadog_checks/kubernetes_state/data/conf.yaml.example#L63
```

Reply:
The idea was to use a `permalink` so when new code is added, the position doesn't change.

Reply:
Ahhh, gotcha, right. Hm. Let me investigate what we usually do, and I'll fix.

Reply:
So, a glimpse behind the curtain: Our support team gets into tricky situations troubleshooting cases where people have been directed (by the docs for example) to well-intentioned-but-outdated versions of config files via links to specific SHAs. Either way, we have a maintenance problem. So, I've worked around it a bit in this fix https://github.com/DataDog/documentation/pull/15154. 

Thank you so much for pointing out the problem with the link!

Reply:
that makes sense 👍🏼

Reply:
better to have the wrong line # than outdated doc